.clang-format

```diff
@@ -0,0 +1,117 @@
+Language: Cpp
+BasedOnStyle: LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignEscapedNewlines: Left
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: true
+AllowShortCaseLabelsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: All
+AllowShortIfStatementsOnASingleLine: true
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: AllDefinitions
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: true
+BinPackParameters: true
+BraceWrapping:
+  AfterClass:            true
+  AfterControlStatement: false
+  AfterEnum:             false
+  AfterFunction:         true
+  AfterNamespace:        false
+  AfterObjCDeclaration:  false
+  AfterStruct:           true
+  AfterUnion:            false
+  AfterExternBlock:      false
+  BeforeCatch:           false
+  BeforeElse:            false
+  IndentBraces:          false
+  SplitEmptyFunction:    true
+  SplitEmptyRecord:      true
+  SplitEmptyNamespace:   false
+BreakBeforeBraces: Custom
+BreakBeforeInheritanceComma: false
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializersBeforeComma: false
+BreakConstructorInitializers: BeforeColon
+BreakAfterJavaFieldAnnotations: false
+BreakBeforeBinaryOperators: NonAssignment
+BreakStringLiterals: true
+ColumnLimit: 100
+CommentPragmas: ''
+CompactNamespaces: false
+ConstructorInitializerAllOnOneLineOrOnePerLine: false
```

Comment:
```suggestion
ConstructorInitializerAllOnOneLineOrOnePerLine: true
```
This results in
```
SyncPoint::SyncPoint(string       identifier,
                     MultiLogger *logger,
                     uint         max_waittime_sec /* = 0 */,
                     uint         max_waittime_nsec /* = 0 */)
: identifier_(identifier),
  emit_calls_(CircularBuffer<SyncPointCall>(1000)),
  wait_for_one_calls_(CircularBuffer<SyncPointCall>(1000)),
  wait_for_all_calls_(CircularBuffer<SyncPointCall>(1000)),
  creation_time_(Time()),
  mutex_(new Mutex()),
  mutex_next_wait_(new Mutex()),
  mutex_wait_for_one_(new Mutex()),
  cond_wait_for_one_(new WaitCondition(mutex_wait_for_one_)),
  mutex_wait_for_all_(new Mutex()),
  cond_wait_for_all_(new WaitCondition(mutex_wait_for_all_)),
  wait_for_all_timer_running_(false),
  max_waittime_sec_(max_waittime_sec),
  max_waittime_nsec_(max_waittime_nsec),
  logger_(logger),
  last_emitter_reset_(Time(0l))
{
```
instead of
```
SyncPoint::SyncPoint(string       identifier,
                     MultiLogger *logger,
                     uint         max_waittime_sec /* = 0 */,
                     uint         max_waittime_nsec /* = 0 */)
: identifier_(identifier), emit_calls_(CircularBuffer<SyncPointCall>(1000)),
  wait_for_one_calls_(CircularBuffer<SyncPointCall>(1000)),
  wait_for_all_calls_(CircularBuffer<SyncPointCall>(1000)), creation_time_(Time()),
  mutex_(new Mutex()), mutex_next_wait_(new Mutex()), mutex_wait_for_one_(new Mutex()),
  cond_wait_for_one_(new WaitCondition(mutex_wait_for_one_)), mutex_wait_for_all_(new Mutex()),
  cond_wait_for_all_(new WaitCondition(mutex_wait_for_all_)), wait_for_all_timer_running_(false),
  max_waittime_sec_(max_waittime_sec), max_waittime_nsec_(max_waittime_nsec), logger_(logger),
  last_emitter_reset_(Time(0l))
{
```